packages/ds-healthcare-gov/src/components/Footer/__snapshots__/Footer.test.tsx.snap

```diff
@@ -273,6 +273,7 @@ exports[`Footer renders basic footer 1`] = `
             <svg
               aria-labelledby="hc-c-footer__logo-title"
               height="25"
+              role="img"
```

Comment:
I'm also noticing now that this was a difference between our icon svgs and these logos. Our icons have this role 👍